### PR TITLE
Rafi's Greeting

### PR DESCRIPTION
**Pull request by rtrad89**
This is the greeting that I want to write, please review and if all OK merge.

### CODE REVIEW

File: rafi greeting.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Howdy y'all peple!
```
Comment (Garywtne) [inline rafi greeting.txt:1]: people

**Review by Garywtne — CHANGES_REQUESTED** [review]
reviewed

File: rafi greeting.txt (hunk 1 above)
Comment (rtrad89) [reply rafi greeting.txt:1]: Oh you're right! right away

**Review by Garywtne — APPROVED** [review]
all ok now

**Pull request merged**